fastlane/Fastfile

```diff
@@ -149,6 +150,10 @@ lane :update_hybrid_common do |options|
     open_pr: options[:open_pr] || false,
     automatic_release: options[:automatic_release] || false
   )
+
+  # Update `yarn.lock` too
+  sh('yarn')
```

Comment:
This is necessary to update this portion too:
```
"@revenuecat/purchases-typescript-internal@npm:8.10.1":
  version: 8.10.1
  resolution: "@revenuecat/purchases-typescript-internal@npm:8.10.1"
  checksum: 1563bb24fd326d5b1cc584b563ea48dbb2a4f50908d8955e704782f8646770e47a74221e89a3dd21733f6fc2201914e406c7aca6cde6fe4fab1ee890cf2189c0
  languageName: node
  linkType: hard
```

Reply:
Hmm does this need to be executed before though? Or maybe add a commit/push steps after this. Otherwise the changes won't be commited/pushed I think

Reply:
I think if we execute it before, the version of phc will still be the same so there won't be any change

I think we have to add the commit and push as you mention

Reply:
Will this fail because `--immutable` is enabled in CI tho?

> If the --immutable option is set (defaults to true on CI), Yarn will abort with an error exit code if the lockfile was to be modified (other paths can be added using the [immutablePatterns](https://yarnpkg.com/configuration/yarnrc#immutablePatterns) configuration setting). For backward compatibility we offer an alias under the name of --frozen-lockfile, but it will be removed in a later release.

Reply:
There's apparently a `--no-immutable` option that's not documented

https://github.com/yarnpkg/berry/discussions/3486#discussioncomment-6214831

Reply:
Thanks! Updated